operator-manifests.catalogsource.yaml

```diff
@@ -1,4 +1,3 @@
-#! validate-crd: ../../deploy/chart/templates/05-catalogsource.crd.yaml
```

Comment:
shouldn't this validate as a catalogsource?

Reply:
Yep, fixed.